cli/src/command/dependency/index.ts

```diff
@@ -25,8 +26,9 @@ export function command(): Command {
   cmd
     .command('push')
     .description('push dependency')
-    .action(() => {
-      push()
+    .option('--no-updatePackage', 'do not update package.json')
```

Comment:
1. --no-updatePackage need rename to --noUpdatePackage
2. option set default value

Reply:
It seems to be a commander's [convention](https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-booleanvalue) to set a `--no` option, which means a negative value of `updatePackage`
`cmd --no-updatePackage` already means `cmd updatePackage=false`, and it can not be set to any other value, so the default is not needed.